frontend/public/components/filter-toolbar.tsx

```diff
@@ -259,16 +280,16 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
   }, []);
 
   const switchFilter = (type: FilterType) => {
-    setFilterType(FilterType[type]);
+    setFilterType(translateFilterType(FilterType[type]));
```

Comment:
Aren't these internal identifiers that shouldn't be translated?

Reply:
If you remove this bit, you wind up with untranslated values in the dropdown (i.e. "Name") when the selection changes.

Reply:
> If you remove this bit, you wind up with untranslated values in the dropdown (i.e. "Name") when the selection changes.

The problem is that we're using the internal identifier as the label in the dropdown. We need two values: an internal identifier and then a mapping to a label that we should in the UI. With this implementation, it will probably break when you switch languages since the value for the wrong language is saved in state.

Reply:
I'll look into adding those this week.

Reply:
All set!